### PR TITLE
update README 

### PR DESCRIPTION
**Pull request by elly0t**

### CODE REVIEW

File: backend/auth/src/main/kotlin/com/oursky/presentation/microservices/kotlin/auth/controller/AuthController.kt [hunk 1]
```diff
@@ -28,7 +28,7 @@ public class AuthController {
         val userId: Long,
         val accessToken: String
     )
-    // curl -X POST http://127.0.0.1:8080/auth/signup -H "Content-Type: application/json" -d '{"user": "test", "pass": "1234"}'
+    // curl -X POST http://127.0.0.1:8080/auth/signup -H "Content-Type: application/json" -d '{"user": "elliot", "pass": "Admin123"}'
```
Comment (hidden-dimensions) [inline backend/auth/src/main/kotlin/com/oursky/presentation/microservices/kotlin/auth/controller/AuthController.kt]: It's a public example, don't personalize it.

File: README.md [hunk 2]
```diff
@@ -1,25 +1,65 @@
 # presentation-microservices-kotlin
-Presentation code and material for micro-services with kotlin
+
+Build microservices wth Kotlin + Spring Boot
```
Comment (hidden-dimensions) [inline README.md]: Move it below the badges, perhaps merge under introduction?

File: README.md [hunk 3]
```diff
@@ -1,25 +1,65 @@
...
+Presented at GDG Kotlin/Everywhere Hong Kong : A Day of Kotlin, 31st Aug 2019
+Slides can be found here
```
Comment (hidden-dimensions) [inline README.md]: missing link to `here`, consider convert it to `slide` file, put on https://github.com/oursky/slides and link here?

File: README.md [hunk 4]
```diff
@@ -1,25 +1,65 @@
...
-##### Start servers
+
+##### Start servers with docker-compose
+
 ```
 cd backend
 docker-compose -p demo up
 ```
+
+##### With Docker
```
Comment (hidden-dimensions) [inline README.md]: docker section was intentionally removed on https://github.com/oursky/presentation-microservices-kotlin/commit/b0c2089c12cf0c4c83db03e558d4a2df767684ab

File: README.md [hunk 5]
```diff
@@ -1,25 +1,65 @@
...
+
+```
+cd backend
+docker build -f Dockerfile.auth . -t demo-auth
+docker run --rm -it -p 8080:8080 -t demo-auth
+# OR
+docker run --rm -it -p 8080:8080 $(docker build -f Dockerfile.auth -q .)
+```
+
+#### Or with on our k8s cluster
```
Comment (hidden-dimensions) [inline README.md]: I don't think we need to encourage access to our demo server. We don't have effort to closely monitor it and we might shut it down any time (and bring it on when needed).

File: README.md [hunk 6]
```diff
@@ -1,25 +1,65 @@
...
+
+Simply curl the APIs by changing the endpoint to `https://present-kotlin.pandawork.com`. Here's an example calling the Sign Up API:
+
+```
+curl -X POST https://present-kotlin.pandawork.com/auth/signup -H "Content-Type: application/json" -d '{"user": "test_user", "pass": "Admin123"}'
+```
+
+#### To see how APIs can be easily created
```
Comment (hidden-dimensions) [inline README.md]: Don't direct reader to individual code, if you want to point it out it should be on the slides.

**Review by hidden-dimensions — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (elly0t) [reply README.md]: ughh a shame, reasonable though

File: README.md (hunk 3 above)
Comment (elly0t) [reply README.md]: can do, let's have a chat over this on Monday?

**Comment by hidden-dimensions** [comment]
merged via https://github.com/oursky/presentation-microservices-kotlin/commit/ae884e0cffc5b0ec30d1b3e97f666fce86096e62

**Pull request closed without merging**